model/variable.cc

```diff
@@ -411,7 +412,8 @@ void VariableBase::draw(cairo_t *cairo) const
   if (VariableValue::isValueId(valueId()))
     vv=minsky::cminsky().variableValues[valueId()];
   
-  if (type()!=constant && !ioVar() && vv.numElements()==1 )
+  // For feature 47
+  if (type()!=constant && !ioVar() && (vv.numDenseElements()==1 || vv.numSparseElements==1) )
```

Comment:
I think we can assume that numSparseElements==1 => numDenseElements()==1, ie basically a scalar variable.

Reply:
The check is now vv.dataSize()==1 only, but I can remove it if necessary.